### PR TITLE
Static Page and Future Translate Base Setup

### PR DESCRIPTION
**Pull request by JOOOEG123**

### CODE REVIEW

File: src/app/core/interceptors/lang.interceptor.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  intercept(
+    request: HttpRequest<unknown>,
+    next: HttpHandler
+  ): Observable<HttpEvent<unknown>> {
+    const lang = localStorage.getItem('lang') || 'en';
+    request = request.clone({
+      setHeaders: {
+        'Accept-Language': lang,
+      },
+    });
```
Comment (JOOOEG123) [inline src/app/core/interceptors/lang.interceptor.ts:23]: future setup

File: src/assets/i18n/cn.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "homepage.component": {
+    "featured_victims_of_the_anti-rightists_movement": "反右运动的特色被害者"
```
Comment (yzhang54) [inline src/assets/i18n/cn.json:3]: Hi, I think some of the translations are quite iffy. Especially "反右运动的特色被害者" and "浏览器".   "项目概况," "画廊," and "文档" might be wrong. I am unsure whether these words will be displayed on the front end. If not, pls ignore my comment.

**Review by yzhang54 — COMMENTED** [review]
Hi,

File: src/assets/i18n/cn.json (hunk 2 above)
Comment (JOOOEG123) [reply src/assets/i18n/cn.json:3]: I will encourage you to highlight each Chinese text and add a comment to what I should replace it with. Please do so as soon as possible. :). Thanks for Noticing I use google translate.

**Comment by JOOOEG123** [comment]
I have added all the changes that Professor Cui requested. Please let me know if you need me to add anymore changes after merging. I can create a separate PR for future fixes. @weilonp @yzhang54 @shanshan-gmu

**Pull request merged**